server/routes/popRoutes.ts

```diff
@@ -12,6 +13,14 @@ export default function routes(): Router {
     router.post(routePath, asyncMiddleware(handler))
   router.use(setUpMultipartFormDataParsing())
 
+  router.use((req, res, next) => {
+    if (req.session.is_pop_login) {
```

Comment:
I'm going to assume this doesn't affect other routes like supervisor - we'll soon see once we include this on other branches